src/d3/scatterplot-ctrls.ts

```diff
@@ -63,8 +63,127 @@ export class Controls{
+        this.scatter.changeScales(
+          this.scatter.getGridBound('x'),
+          this.scatter.getGridBound('y') - 10,
+          this.scatter.getGridBound('z'));
+      }
+      // increase z scale
+      if (event.code === 'Numpad3' || event.code === 'Digit3'){
+      this.scatter.changeScales(
+        this.scatter.getGridBound('x'),
+        this.scatter.getGridBound('y'),
+        this.scatter.getGridBound('z') + 10);
+      }
+      // decrease z scale
+      if (event.code === 'Numpad1' || event.code === 'Digit1'){
+        this.scatter.changeScales(
+          this.scatter.getGridBound('x'),
+          this.scatter.getGridBound('y'),
+          this.scatter.getGridBound('z') - 10);
+      }
+      if (event.code === 'Numpad8' || event.code === 'Digit8'){
+        this.scatter.changeScales(
+          this.scatter.getGridBound('x') + 10,
+          this.scatter.getGridBound('y') + 10,
+          this.scatter.getGridBound('z') + 10);
+      }
+      if (event.code === 'Numpad5' || event.code === 'Digit5'){
+        this.scatter.changeScales(
+          this.scatter.getGridBound('x') - 10,
+          this.scatter.getGridBound('y') - 10,
+          this.scatter.getGridBound('z') - 10);
+      }
+      if (event.code === 'Numpad0' || event.code === 'Digit0'){
+        this.scatter.changeScales(
+          50, 50, 50);
+      }
+      if(event.code === 'KeyT'){
+        const nonTxtControlItems = document.getElementsByClassName('nonTextToggle');
+        for (let item of (nonTxtControlItems as unknown as Array<Element>)){
+          if (!this.showCtrls)
+            (item as AFRAME.Entity).setAttribute('scale', '1 1 1');
+          else
+            (item as AFRAME.Entity).setAttribute('scale', '.01 .01 .01');
+        }
+        const controlItems = document.getElementsByClassName('toggle');
+        for (let item of (controlItems as unknown as Array<Element>)){
+          (item as AFRAME.Entity).setAttribute('visible', !this.showCtrls);
+        }
+        this.showCtrls = !this.showCtrls;
+      }
+      const camPos = document.querySelector('[camera]').object3D.position;
+      if (event.code === 'KeyL'){
+        camPos.set(
+          camPos.x + this.scatter.DAYDREAM_NAV_SPEED,
+          camPos.y,
+          camPos.z
+        );
+      }
+      if (event.code === 'KeyJ'){
+        camPos.set(
+          camPos.x - this.scatter.DAYDREAM_NAV_SPEED,
+          camPos.y,
+          camPos.z
+        );
+      }
+      if (event.code === 'KeyI'){
+        camPos.set(
+          camPos.x,
+          camPos.y,
+          camPos.z  + this.scatter.DAYDREAM_NAV_SPEED
+        );
+      }
+      if (event.code === 'KeyK'){
+        camPos.set(
+          camPos.x,
+          camPos.y,
+          camPos.z - this.scatter.DAYDREAM_NAV_SPEED
+        );
+      }
+      console.log(this.scatter.getGridBound('x'));
+      console.log(this.scatter.getGridBound('y'));
+      console.log(this.scatter.getGridBound('z'));
+    });
+  }
```

Comment:
Should I keep demo keys? They double up as a way to also use the web version. Could be something I add into  my design doc!